pages/1.13/cli/command-reference/dcos-cluster/dcos-cluster-setup/index.md

```diff
@@ -3,63 +3,53 @@ layout: layout.pug
 navigationTitle:  dcos cluster setup
 title: dcos cluster setup
 menuWeight: 6
-excerpt: Configuring a connection to a DC/OS cluster
-
+excerpt: Configuring the CLI to communicate with a cluster
 enterprise: false
 ---
 
 # Description
-
 The `dcos cluster setup` command will configure the connection to a DC/OS cluster, authenticate to DC/OS, and attach to the cluster.
 
 It will also automatically install the [Core and Enterprise CLI plugins](/1.13/cli/plugins/).
 
 # Usage
 
 ```bash
-dcos cluster setup <dcos-url> [OPTIONS]
+  dcos cluster setup <url> [flags]
 ```
 
 # Options
 
-| Name, shorthand | Default | Description |
-|---------|-------------|-------------|
-|  `--ca-certs=<ca-certs>` |             | [enterprise type="inline" size="small" /] The path to a file with trusted CAs to verify requests against.  |
-|  `--insecure` |                        | Allow requests to bypass TLS certificate verification. This is insecure.|
-|  `--no-check` |                        | [enterprise type="inline" size="small" /] Do not check the CA certificate downloaded from the cluster. This is insecure. |
-|  `--password-file=<password_file>`  |  | The path to a file that contains the password for login. |
-|  `--password=<password>`  |            | The password for login on the command line. This is insecure.  |
-|  `--private-key=<key_path>`  |         | The path to a file that contains the private key.  |
-|  `--provider=<provider_id>`  |         | [enterprise type="inline" size="small" /] The authentication provider to use for login.  |
-|  `--username=<username>`  |            | The username for login. |
-|  `--name=<name>`  |            | Specify a custom name for the cluster. |
-|  `--no-plugin`  |            | Do not auto-install dcos-core-cli and dcos-enterprise-cli plugins. |
+| Name | Description |
+|---------|-------------|
+| `--ca-certs string`   |     Specify the path to a file with trusted CAs to verify requests against.
+| `-h`, `--help`     |  Displays help for this command. |
+| `--insecure`       |        Allow requests to bypass TLS certificate verification (insecure).
+| `--name string`     |       Specify a custom name for the cluster.
+| `--no-check`        |       Do not check CA certficate downloaded from cluster (insecure). Applies to Enterprise DC/OS only. [enterprise type="inline" size="small" /]
+| `--password string`   |     Specify the password on the command line (insecure).
+| `--password-file string`  | Specify the path to a file that contains the password.
+| `--private-key string`   |  Specify the path to a file that contains the service account private key.
+| `--provider string`     |   Specify the login provider to use.
+| `--username string`      |  Specify the username for login.
+
```

Comment:
@masonse and @ArmandGrillet are we changing the argument format here from = to blank (this is what it looks like), if yes then this is a big breaking change, and are we allowing older synatx?

Reply:
We accept both, e.g. `dcos auth login --password pwd` and `dcos auth login --password=pwd` are accepted.